### ISSUE TITLE
Python 3.9 changed output of date.isocalendar() from tuple to named tuple

### ISSUE THREAD

**Issue by barsch** [obspy/obspy#2786]
"... basically Python 3.9 changed the output of ```date.isocalendar()``` from ```tuple``` to ```named tuple``` (https://docs.python.org/3/library/datetime.html#datetime.date.isocalendar) - anyway not really an error just needs slight adjustment of the test case"

_Originally posted by @barsch in https://github.com/obspy/obspy/issues/2785#issuecomment-779804501_

We should also state that the output of ```isocalendar()``` etc is the unmodified output of the respective Python ```date``` or ```datetime``` methods.

**Comment by heavelock** [comment]
I wanted to handle this issue in #2722 but to be honest, I am not sure how to properly handle this issue because it is located in doctest.

To be honest, reading through the documentation of doctests (https://docs.python.org/3.8/library/doctest.html#how-it-works), I don't see a nice and clean way of comparing against output from different versions of python. Possibly ellipsis would work but would blur the example itself. I would propose to skip this example and add just annotation about different returns (to output of not executed doctest too).

**Comment by barsch** [comment]
removing the example and just stating different returns depending on the Python version is the way forward imho

**Comment by d-chambers** [comment]
>  I don't see a nice and clean way of comparing against output from different versions of python.

Maybe I am under-thinking it but couldn't we just change these lines

```python
       >>> dt = UTCDateTime(2008, 10, 1, 12, 30, 35, 45020)
        >>> dt.isocalendar()
        (2008, 40, 3)
```

to this?

```python

       >>> dt = UTCDateTime(2008, 10, 1, 12, 30, 35, 45020)
        >>> tuple(dt.isocalendar())
        (2008, 40, 3)
```
Then it should work on all versions.

**Comment by barsch** [comment]
jupp works in 3.8 as well as 3.9 - just tested it

**Comment by barsch** [comment]
fixed - see #2801
